From: Youhei SASAKI <uwabami@gfd-dennou.org>
Date: Thu, 22 May 2014 18:00:40 +0900
Subject: FixMakefileAm

---
 Makefile.am | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- fxlinuxprint.orig/Makefile.am
+++ fxlinuxprint/Makefile.am
@@ -1,10 +1,10 @@
 cupsfilterdir = /usr/lib/cups/filter
-cupsppddir = /usr/share/cups/model/FujiXerox/en
+cupsppddir = /usr/share/ppd/FujiXerox
 cupsmimedir = /etc/cups
 
 cupsfilter_PROGRAMS = pstopdffx pdftopjlfx pdftopdffx
 EXTRA_DIST = fxlinuxprint.ppd mimefx.types mimefx.convs
-cupsppd_DATA = fxlinuxprint.ppd
+cupsppd_DATA = fxlinuxprint.ppd.gz
 cupsmime_DATA = mimefx.types mimefx.convs
 
 pstopdffx_SOURCES = fxlinuxprint.c fxlinuxprint.h codec.c codec.h
